docs/component/testing.md

```diff
@@ -154,6 +154,7 @@ describe('NewMessageForm', () => {
 
     it('calls the send handler', () => {
       expect(sendHandler).toHaveBeenCalledWith(messageText);
+      expect(sendHandler).toHaveBeenCalledTimes(1);
```

Comment:
There is still an addition here that needs to be removed as per my previous request. After that, will be set to merge.

Reply:
Ohh, sorry about that. I missed this one 😅